Classes/Controller/AbstractFluxController.php

```diff
@@ -362,4 +366,41 @@ public function getRecord()
         $row = $this->configurationManager->getContentObject()->data;
         return (array) $row;
     }
+
+    /**
+     * @param array $record Record, automatically mapped from custom TypeConverter which receives only integer UID
+     * @param array $input Array of input from the Form
+     * @return string
+     */
+    public function outletAction(array $record, array $input)
+    {
+        $input['settings'] = $this->settings;
+        try {
+            $outlet = $this->provider->getForm($record)->getOutlet();
+            $outlet->setView($this->view);
```

Comment:
This method assumes ViewAwarePipeInterface, we should have a check for that here.